package.json

```diff
@@ -23,7 +23,7 @@
     "email": "anik220798@gmail.com"
   },
   "peerDependencies": {
-    "stylelint": "^7.0.0 || ^8.0.0"
+    "stylelint": "^13.2.0"
```

Comment:
Breaking change. 

Is this causing any issue if we don't update this ? I not sure about this,

Reply:
I use `stylelint@"^13.12.0"` and I get this error message.

```
npm ERR! Could not resolve dependency:
npm ERR! peer stylelint@"^7.0.0 || ^8.0.0" from stylelint-postcss-ignore@0.1.0
npm ERR! node_modules/stylelint-postcss-ignore
npm ERR!   stylelint-postcss-ignore@"anikethsaha/stylelint-postcss-ignore#master" from the root project
```